stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/MessageListView.kt

```diff
@@ -377,14 +378,15 @@ public class MessageListView : ConstraintLayout {
     private fun init(context: Context, attr: AttributeSet?) {
         messageListViewStyle = MessageListViewStyle(context, attr)
 
-        binding = StreamUiMessageListViewBinding.inflate(context.inflater, this, true)
+        binding = StreamUiMessageListViewBinding.inflate(context.inflater, this)
 
         initRecyclerView()
         initScrollHelper()
         initLoadingView()
         initEmptyStateView()
 
         configureAttributes(attr)
+        layoutTransition = LayoutTransition()
```

Comment:
Why is this needed?

Reply:
to keep the behaviour, previously in the XML there was a `android:animateLayoutChanges=true`. This is the programatic approach to do it. 
Personally I would be fine removing it.

Reply:
Makes sense. Let's keep it.